### PR TITLE
Update signature-update-1.png

### PR DESCRIPTION
**Pull request by msbemba**
made changes to the screenshot as the options have changed

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7067

File: microsoft-365/security/defender-endpoint/configure-endpoints-gp.md [hunk 1]
```diff
@@ -299,7 +299,7 @@ Browse to **Computer Configuration** \> **Policies** \> **Administrative Templat
 
 ### Check for signature update
 
-Browse to **Computer Configuration** \> **Policies** \> **Administrative Templates** \> **Windows Components** \> **Microsoft Defender Antivirus** \> **Signature Updates**
+Browse to **Computer Configuration** \> **Policies** \> **Administrative Templates** \> **Windows Components** \> **Microsoft Defender Antivirus** \> **Security Intelligence Updates*
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-gp.md]:
```suggestion
Browse to **Computer Configuration** \> **Policies** \> **Administrative Templates** \> **Windows Components** \> **Microsoft Defender Antivirus** \> **Security Intelligence Updates**.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**